package.json

```diff
@@ -27,39 +27,39 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "async": "^1.4.0",
-    "babel": "^5.8.20",
-    "babel-core": "^5.8.20",
+    "async": "^1.4.2",
+    "babel": "^5.8.23",
+    "babel-core": "^5.8.25",
     "babel-loader": "^5.3.2",
-    "chai": "^3.2.0",
-    "colors": "^1.1.0",
-    "eslint": "^1.0.0",
+    "chai": "^3.3.0",
+    "colors": "^1.1.2",
+    "eslint": "^1.6.0",
     "grunt": "^0.4.5",
-    "grunt-babel": "^5.0.1",
+    "grunt-babel": "^5.0.3",
     "grunt-clean": "^0.4.0",
     "grunt-cli": "^0.1.13",
     "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-copy": "^0.8.0",
-    "grunt-contrib-uglify": "^0.9.1",
+    "grunt-contrib-copy": "^0.8.1",
+    "grunt-contrib-uglify": "^0.9.2",
     "grunt-contrib-watch": "^0.6.1",
-    "grunt-eslint": "^17.0.0",
-    "grunt-karma": "^0.12.0",
-    "grunt-mocha-istanbul": "^2.4.0",
+    "grunt-eslint": "^17.3.1",
+    "grunt-karma": "^0.12.1",
+    "grunt-mocha-istanbul": "^3.0.1",
     "grunt-mocha-test": "^0.12.7",
     "grunt-webpack": "^1.0.11",
-    "istanbul": "^0.3.2",
-    "karma": "^0.13.8",
+    "istanbul": "^0.3.22",
+    "karma": "^0.13.11",
     "karma-mocha": "^0.2.0",
     "karma-mocha-reporter": "^1.1.1",
     "karma-phantomjs-launcher": "^0.2.1",
     "karma-sauce-launcher": "^0.2.14",
-    "karma-sourcemap-loader": "^0.3.5",
+    "karma-sourcemap-loader": "^0.3.6",
     "karma-webpack": "^1.7.0",
-    "mocha": "^2.2.4",
-    "phantomjs": "^1.9.17",
-    "semver": "^5.0.1",
-    "webpack": "^1.10.5",
-    "webpack-dev-server": "^1.10.1"
+    "mocha": "^2.3.3",
+    "phantomjs": "^1.9.18",
+    "semver": "^5.0.3",
+    "webpack": "^1.12.2",
+    "webpack-dev-server": "^1.12.0"
```

Comment:
As long as the tests pass I'm fine taking version changes as is, but generally it's better to keep the changes in a PR down to just those needed to implement your feature. Makes it easier to review and less likely to be rejected for reasons that are tangential to your feature change.